src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-dialog.js

```diff
@@ -0,0 +1,277 @@
+                        [IDENTICAL_SECTIONS]:
+                            shuntCompensator.isIdenticalSection,
+                        [SUSCEPTANCE_PER_SECTION]:
+                            shuntCompensator.susceptancePerSection,
+                        ...getConnectivityFormData({
+                            voltageLevelId: shuntCompensator.voltageLevelId,
+                            voltageLevelTopologyKind: vlResult.topologyKind,
+                            voltageLevelName: vlResult.name,
+                            voltageLevelNominalVoltage: vlResult.nominalVoltage,
+                            voltageLevelSubstationId: vlResult.substationId,
+                            busbarSectionId:
+                                shuntCompensator.busOrBusbarSectionId,
+                            connectionDirection:
+                                shuntCompensator.connectionDirection,
+                            connectionName: shuntCompensator.connectionName,
+                            connectionPosition:
+                                shuntCompensator.connectionPosition,
+                        }),
+                    });
+                }) // if voltage level can't be found, we fill the form with minimal infos
+                .catch(() => {
+                    reset({
+                        [EQUIPMENT_ID]: shuntCompensator.equipmentId,
+                        [EQUIPMENT_NAME]: shuntCompensator.equipmentName ?? '',
+                        [MAXIMUM_NUMBER_OF_SECTIONS]:
+                            shuntCompensator.maximumNumberOfSections,
+                        [CURRENT_NUMBER_OF_SECTIONS]:
+                            shuntCompensator.currentNumberOfSections,
+                        [IDENTICAL_SECTIONS]:
+                            shuntCompensator.isIdenticalSection,
+                        [SUSCEPTANCE_PER_SECTION]:
+                            shuntCompensator.susceptancePerSection,
+                        ...getConnectivityFormData({
+                            voltageLevelId: shuntCompensator.voltageLevelId,
+                            busbarSectionId:
+                                shuntCompensator.busOrBusbarSectionId,
+                            connectionDirection:
+                                shuntCompensator.connectionDirection,
+                            connectionName: shuntCompensator.connectionName,
+                            connectionPosition:
+                                shuntCompensator.connectionPosition,
+                        }),
+                    });
+                });
+        },
+        [studyUuid, currentNodeUuid, reset]
+    );
+
+    const searchCopy = useFormSearchCopy({
+        studyUuid,
+        currentNodeUuid,
+        equipmentPath,
+        toFormValues: (data) => data,
+        setFormValues: fromSearchCopyToFormValues,
+    });
+
+    useEffect(() => {
+        if (editData) {
+            fromEditDataToFormValues(editData);
```

Comment:
when we click edit button after create hypothesis of 'shunt componsator', the validate button is not disabled

Reply:
fixed